html_css/intermediate_css/selectors.md

```diff
@@ -89,6 +89,8 @@ Before diving into pseudo-selectors, a quick note on the difference between [pse
 ### Pseudo-classes
 Pseudo-classes offer us different ways to target elements in our HTML. There are quite a lot of them, and they come in a couple of different flavors. Some are based on their position or structure within the HTML. Others are based on the state of a particular element, or how the user is currently interacting with it. There are too many to cover in detail here but we'll have a look at some of the most useful ones. Pseudo-classes share the same specificity as regular classes (0, 0, 1, 0). Just like regular classes, most can be chained together
 
+**Note:** The (0,0,1,0) above is the notation for calculating specificity. You can find more about how it works [here](https://slicejack.com/quick-guide-to-css-specificity/)
```

Comment:
@codyloyd do you have any feedback on the quality of this resource?

One thing I would like changed if we do move forward with this PR, the link text used could be improved for accessibility. Check out [this article about link text and accessibility by WebAIM](https://webaim.org/techniques/hypertext/#screen-readers-and-links) for more information about the topic.

Reply:
> @codyloyd do you have any feedback on the quality of this resource?
> 
> One thing I would like changed if we do move forward with this PR, the link text used could be improved for accessibility. Check out [this article about link text and accessibility by WebAIM](https://webaim.org/techniques/hypertext/#screen-readers-and-links) for more information about the topic.

Just want to chime in here... As noted in the comments of that article, some of the content was stolen word for word from CSS-Tricks. The actual CSS-Tricks article on specificity is here (https://css-tricks.com/specifics-on-css-specificity/) and might be a better resource. CSS-Tricks is already used throughout our CSS curriculum.

Reply:
That article seems great too! I was trying to find a resource which just went over the notation instead of going too deep into specificity